### PR TITLE
Fix socket resp handling

### PR DESCRIPTION
**Pull request by krish2718**

### CODE REVIEW

**Review by kapbh — APPROVED** [review]

File: wpa_supplicant/wpa_cli_zephyr.c [hunk 1]
```diff
@@ -76,7 +76,8 @@ static int _wpa_ctrl_command(struct wpa_ctrl *ctrl, const char *cmd, int print,
 
 	if (print) {
 			buf[len] = '\0';
-			printf("%s", buf);
+			if (strlen(buf) > 0)
```
Comment (jukkar) [inline wpa_supplicant/wpa_cli_zephyr.c:79]: Minor nit but we could shorten this to
```
if (buf[0] != '\0')
    printf("%s", buf);
```
This avoids a call to `strlen()`

**Review by jukkar — APPROVED** [review]
Just a mental note to the team, we need to remember to port these to upstream when the time comes.
